.travis.yml

```diff
@@ -1,8 +1,6 @@
 language: scala
 scala:
-   - "2.9.2"
+   - "2.10.1"
```

Comment:
Good point. Shouldn't matter to much for a minor version, but I change it nonetheless.